!SESSION 2017-01-09 10:57:16.402 -----------------------------------------------
eclipse.buildId=4.6.1.M20160907-1200
java.version=1.8.0_111
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.jee.product
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.jee.product

This is a continuation of log file E:\eclipse\student\.metadata\.bak_0.log
Created Time: 2017-01-09 11:06:24.987

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:06:24.991
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:06:25.064
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:06:25.585
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:06:25.722
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:06:26.586
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:06:26.643
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:06:26.693
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:06:29.789
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:06:29.973
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:06:30.028
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:06:30.088
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:06:30.174
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:06:30.230
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:06:42.978
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:06:43.034
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:07:00.936
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:07:01.004
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:07:02.274
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:07:02.354
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:07:09.040
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:07:10.768
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:07:10.989
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:07:11.045
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:07:11.104
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:07:13.083
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:07:13.379
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:07:13.425
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:07:54.335
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:07:56.927
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:08:00.053
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:08:09.582
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:08:19.368
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:08:20.707
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:08:20.802
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:08:20.882
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:08:20.961
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:08:29.049
!MESSAGE Unable to retrieve a list of remote deployment scanners
!STACK 0
org.jboss.ide.eclipse.as.management.core.JBoss7ManangerException: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:361)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9ManagerService.execute(Wildfly9ManagerService.java:171)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerServiceProxy.execute(JBoss7ManagerServiceProxy.java:87)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:294)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:1)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerUtil.executeWithService(JBoss7ManagerUtil.java:136)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.executeWithResult(AS7DeploymentScannerUtility.java:292)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScanners(AS7DeploymentScannerUtility.java:163)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScannersFromServer(AS7DeploymentScannerUtility.java:252)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServer(LocalJBoss7DeploymentScannerAdditions.java:173)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServerSafely(LocalJBoss7DeploymentScannerAdditions.java:135)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.ensureScannersRemoved(LocalJBoss7DeploymentScannerAdditions.java:150)
	at org.jboss.ide.eclipse.as.core.server.internal.AbstractDeploymentScannerAdditions.removeAddedDeploymentScanners(AbstractDeploymentScannerAdditions.java:82)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.removeScanners(LocalLegacyShutdownController.java:75)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:58)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.ide.eclipse.as.wtp.core.server.behavior.ControllableServerBehavior.stop(ControllableServerBehavior.java:255)
	at org.eclipse.wst.server.core.internal.Server.stopImpl2(Server.java:3688)
	at org.eclipse.wst.server.core.internal.Server.stopImpl(Server.java:3645)
	at org.eclipse.wst.server.core.internal.Server$StopJob.run(Server.java:403)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:149)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:75)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:350)
	... 20 more
Caused by: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:122)
	at org.jboss.as.protocol.ProtocolConnectionManager$EstablishingConnection.connect(ProtocolConnectionManager.java:257)
	at org.jboss.as.protocol.ProtocolConnectionManager.connect(ProtocolConnectionManager.java:71)
	at org.jboss.as.protocol.mgmt.FutureManagementChannel$Establishing.getChannel(FutureManagementChannel.java:212)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient.getOrCreateChannel(RemotingModelControllerClient.java:146)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient$1.getChannel(RemotingModelControllerClient.java:65)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:147)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:122)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeRequest(AbstractModelControllerClient.java:263)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:168)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:147)
	... 22 more
Caused by: java.io.IOException: Invalid response
	at org.xnio.http.HttpUpgradeParser.parseVersion(HttpUpgradeParser.java:157)
	at org.xnio.http.HttpUpgradeParser.parse(HttpUpgradeParser.java:55)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:419)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:400)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:88)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:559)
	at ...asynchronous invocation...(Unknown Source)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:294)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:275)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:379)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:367)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connect(ProtocolConnectionUtils.java:83)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:114)
	... 32 more

!ENTRY org.eclipse.wst.server.core 4 0 2017-01-09 11:08:29.160
!MESSAGE Server WildFly 10.x failed to stop.

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:08:32.974
!MESSAGE Unable to retrieve a list of remote deployment scanners
!STACK 0
org.jboss.ide.eclipse.as.management.core.JBoss7ManangerException: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:361)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9ManagerService.execute(Wildfly9ManagerService.java:171)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerServiceProxy.execute(JBoss7ManagerServiceProxy.java:87)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:294)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:1)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerUtil.executeWithService(JBoss7ManagerUtil.java:136)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.executeWithResult(AS7DeploymentScannerUtility.java:292)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScanners(AS7DeploymentScannerUtility.java:163)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScannersFromServer(AS7DeploymentScannerUtility.java:252)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServer(LocalJBoss7DeploymentScannerAdditions.java:173)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServerSafely(LocalJBoss7DeploymentScannerAdditions.java:135)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.ensureScannersRemoved(LocalJBoss7DeploymentScannerAdditions.java:150)
	at org.jboss.ide.eclipse.as.core.server.internal.AbstractDeploymentScannerAdditions.removeAddedDeploymentScanners(AbstractDeploymentScannerAdditions.java:82)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.removeScanners(LocalLegacyShutdownController.java:75)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:58)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.ide.eclipse.as.wtp.core.server.behavior.ControllableServerBehavior.stop(ControllableServerBehavior.java:255)
	at org.eclipse.wst.server.core.internal.Server.stopImpl2(Server.java:3688)
	at org.eclipse.wst.server.core.internal.Server.stopImpl(Server.java:3645)
	at org.eclipse.wst.server.core.internal.Server$StopJob.run(Server.java:403)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:149)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:75)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:350)
	... 20 more
Caused by: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:122)
	at org.jboss.as.protocol.ProtocolConnectionManager$EstablishingConnection.connect(ProtocolConnectionManager.java:257)
	at org.jboss.as.protocol.ProtocolConnectionManager.connect(ProtocolConnectionManager.java:71)
	at org.jboss.as.protocol.mgmt.FutureManagementChannel$Establishing.getChannel(FutureManagementChannel.java:212)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient.getOrCreateChannel(RemotingModelControllerClient.java:146)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient$1.getChannel(RemotingModelControllerClient.java:65)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:147)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:122)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeRequest(AbstractModelControllerClient.java:263)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:168)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:147)
	... 22 more
Caused by: java.io.IOException: Invalid response
	at org.xnio.http.HttpUpgradeParser.parseVersion(HttpUpgradeParser.java:157)
	at org.xnio.http.HttpUpgradeParser.parse(HttpUpgradeParser.java:55)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:419)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:400)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:88)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:559)
	at ...asynchronous invocation...(Unknown Source)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:294)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:275)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:379)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:367)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connect(ProtocolConnectionUtils.java:83)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:114)
	... 32 more

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:08:33.069
!MESSAGE Unable to retrieve a list of remote deployment scanners
!STACK 0
org.jboss.ide.eclipse.as.management.core.JBoss7ManangerException: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:361)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9ManagerService.execute(Wildfly9ManagerService.java:171)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerServiceProxy.execute(JBoss7ManagerServiceProxy.java:87)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:294)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:1)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerUtil.executeWithService(JBoss7ManagerUtil.java:136)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.executeWithResult(AS7DeploymentScannerUtility.java:292)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScanners(AS7DeploymentScannerUtility.java:163)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScannersFromServer(AS7DeploymentScannerUtility.java:252)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServer(LocalJBoss7DeploymentScannerAdditions.java:173)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServerSafely(LocalJBoss7DeploymentScannerAdditions.java:135)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.ensureScannersRemoved(LocalJBoss7DeploymentScannerAdditions.java:150)
	at org.jboss.ide.eclipse.as.core.server.internal.AbstractDeploymentScannerAdditions.removeAddedDeploymentScanners(AbstractDeploymentScannerAdditions.java:82)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.removeScanners(LocalLegacyShutdownController.java:75)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:58)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.ManagementShutdownController.forceStop(ManagementShutdownController.java:63)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stopImpl(LocalLegacyShutdownController.java:120)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:66)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.ide.eclipse.as.wtp.core.server.behavior.ControllableServerBehavior.stop(ControllableServerBehavior.java:255)
	at org.eclipse.wst.server.core.internal.Server.stopImpl2(Server.java:3688)
	at org.eclipse.wst.server.core.internal.Server.stopImpl(Server.java:3645)
	at org.eclipse.wst.server.core.internal.Server$StopJob.run(Server.java:403)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:149)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:75)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:350)
	... 24 more
Caused by: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:122)
	at org.jboss.as.protocol.ProtocolConnectionManager$EstablishingConnection.connect(ProtocolConnectionManager.java:257)
	at org.jboss.as.protocol.ProtocolConnectionManager.connect(ProtocolConnectionManager.java:71)
	at org.jboss.as.protocol.mgmt.FutureManagementChannel$Establishing.getChannel(FutureManagementChannel.java:212)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient.getOrCreateChannel(RemotingModelControllerClient.java:146)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient$1.getChannel(RemotingModelControllerClient.java:65)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:147)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:122)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeRequest(AbstractModelControllerClient.java:263)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:168)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:147)
	... 26 more
Caused by: java.io.IOException: Invalid response
	at org.xnio.http.HttpUpgradeParser.parseVersion(HttpUpgradeParser.java:157)
	at org.xnio.http.HttpUpgradeParser.parse(HttpUpgradeParser.java:55)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:419)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:400)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:88)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:559)
	at ...asynchronous invocation...(Unknown Source)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:294)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:275)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:379)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:367)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connect(ProtocolConnectionUtils.java:83)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:114)
	... 36 more

!ENTRY org.jboss.ide.eclipse.as.core 4 16973826 2017-01-09 11:08:33.097
!MESSAGE The server was shutdown forcefully. All processes were terminated.

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:08:37.364
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:08:37.429
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:08:53.563
!MESSAGE The server "WildFly 10.x" requires a JDK to run properly. The VM used to launch this server is not recognized as a valid JDK. C:\Program Files\Java\jre1.8.0_111 

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:09:08.098
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:09:08.113
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:10:51.595
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:10:51.778
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:11:17.147
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:11:26.749
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:11:26.813
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:11:30.610
!MESSAGE Unable to retrieve a list of remote deployment scanners
!STACK 0
org.jboss.ide.eclipse.as.management.core.JBoss7ManangerException: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:361)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9ManagerService.execute(Wildfly9ManagerService.java:171)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerServiceProxy.execute(JBoss7ManagerServiceProxy.java:87)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:294)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:1)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerUtil.executeWithService(JBoss7ManagerUtil.java:136)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.executeWithResult(AS7DeploymentScannerUtility.java:292)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScanners(AS7DeploymentScannerUtility.java:163)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScannersFromServer(AS7DeploymentScannerUtility.java:252)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServer(LocalJBoss7DeploymentScannerAdditions.java:173)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServerSafely(LocalJBoss7DeploymentScannerAdditions.java:135)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.ensureScannersRemoved(LocalJBoss7DeploymentScannerAdditions.java:150)
	at org.jboss.ide.eclipse.as.core.server.internal.AbstractDeploymentScannerAdditions.removeAddedDeploymentScanners(AbstractDeploymentScannerAdditions.java:82)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.removeScanners(LocalLegacyShutdownController.java:75)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:58)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.ide.eclipse.as.wtp.core.server.behavior.ControllableServerBehavior.stop(ControllableServerBehavior.java:255)
	at org.eclipse.wst.server.core.internal.Server.stopImpl2(Server.java:3688)
	at org.eclipse.wst.server.core.internal.Server.stopImpl(Server.java:3645)
	at org.eclipse.wst.server.core.internal.Server$StopJob.run(Server.java:403)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:149)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:75)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:350)
	... 20 more
Caused by: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:122)
	at org.jboss.as.protocol.ProtocolConnectionManager$EstablishingConnection.connect(ProtocolConnectionManager.java:257)
	at org.jboss.as.protocol.ProtocolConnectionManager.connect(ProtocolConnectionManager.java:71)
	at org.jboss.as.protocol.mgmt.FutureManagementChannel$Establishing.getChannel(FutureManagementChannel.java:212)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient.getOrCreateChannel(RemotingModelControllerClient.java:146)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient$1.getChannel(RemotingModelControllerClient.java:65)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:147)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:122)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeRequest(AbstractModelControllerClient.java:263)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:168)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:147)
	... 22 more
Caused by: java.io.IOException: Invalid response
	at org.xnio.http.HttpUpgradeParser.parseVersion(HttpUpgradeParser.java:157)
	at org.xnio.http.HttpUpgradeParser.parse(HttpUpgradeParser.java:55)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:419)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:400)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:88)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:559)
	at ...asynchronous invocation...(Unknown Source)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:294)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:275)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:379)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:367)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connect(ProtocolConnectionUtils.java:83)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:114)
	... 32 more

!ENTRY org.eclipse.wst.server.core 4 0 2017-01-09 11:11:30.723
!MESSAGE Server WildFly 10.x failed to stop.

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:11:34.277
!MESSAGE Unable to retrieve a list of remote deployment scanners
!STACK 0
org.jboss.ide.eclipse.as.management.core.JBoss7ManangerException: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:361)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9ManagerService.execute(Wildfly9ManagerService.java:171)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerServiceProxy.execute(JBoss7ManagerServiceProxy.java:87)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:294)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:1)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerUtil.executeWithService(JBoss7ManagerUtil.java:136)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.executeWithResult(AS7DeploymentScannerUtility.java:292)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScanners(AS7DeploymentScannerUtility.java:163)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScannersFromServer(AS7DeploymentScannerUtility.java:252)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServer(LocalJBoss7DeploymentScannerAdditions.java:173)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServerSafely(LocalJBoss7DeploymentScannerAdditions.java:135)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.ensureScannersRemoved(LocalJBoss7DeploymentScannerAdditions.java:150)
	at org.jboss.ide.eclipse.as.core.server.internal.AbstractDeploymentScannerAdditions.removeAddedDeploymentScanners(AbstractDeploymentScannerAdditions.java:82)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.removeScanners(LocalLegacyShutdownController.java:75)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:58)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.ide.eclipse.as.wtp.core.server.behavior.ControllableServerBehavior.stop(ControllableServerBehavior.java:255)
	at org.eclipse.wst.server.core.internal.Server.stopImpl2(Server.java:3688)
	at org.eclipse.wst.server.core.internal.Server.stopImpl(Server.java:3645)
	at org.eclipse.wst.server.core.internal.Server$StopJob.run(Server.java:403)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:149)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:75)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:350)
	... 20 more
Caused by: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:122)
	at org.jboss.as.protocol.ProtocolConnectionManager$EstablishingConnection.connect(ProtocolConnectionManager.java:257)
	at org.jboss.as.protocol.ProtocolConnectionManager.connect(ProtocolConnectionManager.java:71)
	at org.jboss.as.protocol.mgmt.FutureManagementChannel$Establishing.getChannel(FutureManagementChannel.java:212)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient.getOrCreateChannel(RemotingModelControllerClient.java:146)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient$1.getChannel(RemotingModelControllerClient.java:65)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:147)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:122)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeRequest(AbstractModelControllerClient.java:263)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:168)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:147)
	... 22 more
Caused by: java.io.IOException: Invalid response
	at org.xnio.http.HttpUpgradeParser.parseVersion(HttpUpgradeParser.java:157)
	at org.xnio.http.HttpUpgradeParser.parse(HttpUpgradeParser.java:55)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:419)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:400)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:88)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:559)
	at ...asynchronous invocation...(Unknown Source)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:294)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:275)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:379)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:367)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connect(ProtocolConnectionUtils.java:83)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:114)
	... 32 more

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:11:34.352
!MESSAGE Unable to retrieve a list of remote deployment scanners
!STACK 0
org.jboss.ide.eclipse.as.management.core.JBoss7ManangerException: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:361)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9ManagerService.execute(Wildfly9ManagerService.java:171)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerServiceProxy.execute(JBoss7ManagerServiceProxy.java:87)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:294)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:1)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerUtil.executeWithService(JBoss7ManagerUtil.java:136)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.executeWithResult(AS7DeploymentScannerUtility.java:292)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScanners(AS7DeploymentScannerUtility.java:163)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScannersFromServer(AS7DeploymentScannerUtility.java:252)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServer(LocalJBoss7DeploymentScannerAdditions.java:173)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServerSafely(LocalJBoss7DeploymentScannerAdditions.java:135)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.ensureScannersRemoved(LocalJBoss7DeploymentScannerAdditions.java:150)
	at org.jboss.ide.eclipse.as.core.server.internal.AbstractDeploymentScannerAdditions.removeAddedDeploymentScanners(AbstractDeploymentScannerAdditions.java:82)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.removeScanners(LocalLegacyShutdownController.java:75)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:58)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.ManagementShutdownController.forceStop(ManagementShutdownController.java:63)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stopImpl(LocalLegacyShutdownController.java:120)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:66)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.ide.eclipse.as.wtp.core.server.behavior.ControllableServerBehavior.stop(ControllableServerBehavior.java:255)
	at org.eclipse.wst.server.core.internal.Server.stopImpl2(Server.java:3688)
	at org.eclipse.wst.server.core.internal.Server.stopImpl(Server.java:3645)
	at org.eclipse.wst.server.core.internal.Server$StopJob.run(Server.java:403)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:149)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:75)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:350)
	... 24 more
Caused by: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:122)
	at org.jboss.as.protocol.ProtocolConnectionManager$EstablishingConnection.connect(ProtocolConnectionManager.java:257)
	at org.jboss.as.protocol.ProtocolConnectionManager.connect(ProtocolConnectionManager.java:71)
	at org.jboss.as.protocol.mgmt.FutureManagementChannel$Establishing.getChannel(FutureManagementChannel.java:212)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient.getOrCreateChannel(RemotingModelControllerClient.java:146)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient$1.getChannel(RemotingModelControllerClient.java:65)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:147)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:122)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeRequest(AbstractModelControllerClient.java:263)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:168)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:147)
	... 26 more
Caused by: java.io.IOException: Invalid response
	at org.xnio.http.HttpUpgradeParser.parseVersion(HttpUpgradeParser.java:157)
	at org.xnio.http.HttpUpgradeParser.parse(HttpUpgradeParser.java:55)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:419)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:400)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:88)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:559)
	at ...asynchronous invocation...(Unknown Source)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:294)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:275)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:379)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:367)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connect(ProtocolConnectionUtils.java:83)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:114)
	... 36 more

!ENTRY org.jboss.ide.eclipse.as.core 4 16973826 2017-01-09 11:11:34.375
!MESSAGE The server was shutdown forcefully. All processes were terminated.

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:11:38.953
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:11:38.976
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:11:50.757
!MESSAGE The server "WildFly 10.x" requires a JDK to run properly. The VM used to launch this server is not recognized as a valid JDK. C:\Program Files\Java\jre1.8.0_111 

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:12:04.815
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:12:04.828
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:13:59.189
!MESSAGE Unable to retrieve a list of remote deployment scanners
!STACK 0
org.jboss.ide.eclipse.as.management.core.JBoss7ManangerException: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:361)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9ManagerService.execute(Wildfly9ManagerService.java:171)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerServiceProxy.execute(JBoss7ManagerServiceProxy.java:87)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:294)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:1)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerUtil.executeWithService(JBoss7ManagerUtil.java:136)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.executeWithResult(AS7DeploymentScannerUtility.java:292)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScanners(AS7DeploymentScannerUtility.java:163)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScannersFromServer(AS7DeploymentScannerUtility.java:252)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServer(LocalJBoss7DeploymentScannerAdditions.java:173)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServerSafely(LocalJBoss7DeploymentScannerAdditions.java:135)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.ensureScannersRemoved(LocalJBoss7DeploymentScannerAdditions.java:150)
	at org.jboss.ide.eclipse.as.core.server.internal.AbstractDeploymentScannerAdditions.removeAddedDeploymentScanners(AbstractDeploymentScannerAdditions.java:82)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.removeScanners(LocalLegacyShutdownController.java:75)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:58)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.ide.eclipse.as.wtp.core.server.behavior.ControllableServerBehavior.stop(ControllableServerBehavior.java:255)
	at org.eclipse.wst.server.core.internal.Server.stopImpl2(Server.java:3688)
	at org.eclipse.wst.server.core.internal.Server.stopImpl(Server.java:3645)
	at org.eclipse.wst.server.core.internal.Server$StopJob.run(Server.java:403)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:149)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:75)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:350)
	... 20 more
Caused by: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:122)
	at org.jboss.as.protocol.ProtocolConnectionManager$EstablishingConnection.connect(ProtocolConnectionManager.java:257)
	at org.jboss.as.protocol.ProtocolConnectionManager.connect(ProtocolConnectionManager.java:71)
	at org.jboss.as.protocol.mgmt.FutureManagementChannel$Establishing.getChannel(FutureManagementChannel.java:212)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient.getOrCreateChannel(RemotingModelControllerClient.java:146)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient$1.getChannel(RemotingModelControllerClient.java:65)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:147)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:122)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeRequest(AbstractModelControllerClient.java:263)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:168)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:147)
	... 22 more
Caused by: java.io.IOException: Invalid response
	at org.xnio.http.HttpUpgradeParser.parseVersion(HttpUpgradeParser.java:157)
	at org.xnio.http.HttpUpgradeParser.parse(HttpUpgradeParser.java:55)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:419)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:400)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:88)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:559)
	at ...asynchronous invocation...(Unknown Source)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:294)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:275)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:379)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:367)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connect(ProtocolConnectionUtils.java:83)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:114)
	... 32 more

!ENTRY org.eclipse.wst.server.core 4 0 2017-01-09 11:13:59.273
!MESSAGE Server WildFly 10.x failed to stop.

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:14:03.355
!MESSAGE Unable to retrieve a list of remote deployment scanners
!STACK 0
org.jboss.ide.eclipse.as.management.core.JBoss7ManangerException: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:361)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9ManagerService.execute(Wildfly9ManagerService.java:171)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerServiceProxy.execute(JBoss7ManagerServiceProxy.java:87)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:294)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:1)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerUtil.executeWithService(JBoss7ManagerUtil.java:136)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.executeWithResult(AS7DeploymentScannerUtility.java:292)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScanners(AS7DeploymentScannerUtility.java:163)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScannersFromServer(AS7DeploymentScannerUtility.java:252)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServer(LocalJBoss7DeploymentScannerAdditions.java:173)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServerSafely(LocalJBoss7DeploymentScannerAdditions.java:135)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.ensureScannersRemoved(LocalJBoss7DeploymentScannerAdditions.java:150)
	at org.jboss.ide.eclipse.as.core.server.internal.AbstractDeploymentScannerAdditions.removeAddedDeploymentScanners(AbstractDeploymentScannerAdditions.java:82)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.removeScanners(LocalLegacyShutdownController.java:75)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:58)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.ide.eclipse.as.wtp.core.server.behavior.ControllableServerBehavior.stop(ControllableServerBehavior.java:255)
	at org.eclipse.wst.server.core.internal.Server.stopImpl2(Server.java:3688)
	at org.eclipse.wst.server.core.internal.Server.stopImpl(Server.java:3645)
	at org.eclipse.wst.server.core.internal.Server$StopJob.run(Server.java:403)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:149)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:75)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:350)
	... 20 more
Caused by: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:122)
	at org.jboss.as.protocol.ProtocolConnectionManager$EstablishingConnection.connect(ProtocolConnectionManager.java:257)
	at org.jboss.as.protocol.ProtocolConnectionManager.connect(ProtocolConnectionManager.java:71)
	at org.jboss.as.protocol.mgmt.FutureManagementChannel$Establishing.getChannel(FutureManagementChannel.java:212)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient.getOrCreateChannel(RemotingModelControllerClient.java:146)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient$1.getChannel(RemotingModelControllerClient.java:65)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:147)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:122)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeRequest(AbstractModelControllerClient.java:263)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:168)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:147)
	... 22 more
Caused by: java.io.IOException: Invalid response
	at org.xnio.http.HttpUpgradeParser.parseVersion(HttpUpgradeParser.java:157)
	at org.xnio.http.HttpUpgradeParser.parse(HttpUpgradeParser.java:55)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:419)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:400)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:88)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:559)
	at ...asynchronous invocation...(Unknown Source)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:294)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:275)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:379)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:367)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connect(ProtocolConnectionUtils.java:83)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:114)
	... 32 more

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:14:03.427
!MESSAGE Unable to retrieve a list of remote deployment scanners
!STACK 0
org.jboss.ide.eclipse.as.management.core.JBoss7ManangerException: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:361)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9ManagerService.execute(Wildfly9ManagerService.java:171)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerServiceProxy.execute(JBoss7ManagerServiceProxy.java:87)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:294)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:1)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerUtil.executeWithService(JBoss7ManagerUtil.java:136)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.executeWithResult(AS7DeploymentScannerUtility.java:292)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScanners(AS7DeploymentScannerUtility.java:163)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScannersFromServer(AS7DeploymentScannerUtility.java:252)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServer(LocalJBoss7DeploymentScannerAdditions.java:173)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServerSafely(LocalJBoss7DeploymentScannerAdditions.java:135)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.ensureScannersRemoved(LocalJBoss7DeploymentScannerAdditions.java:150)
	at org.jboss.ide.eclipse.as.core.server.internal.AbstractDeploymentScannerAdditions.removeAddedDeploymentScanners(AbstractDeploymentScannerAdditions.java:82)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.removeScanners(LocalLegacyShutdownController.java:75)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:58)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.ManagementShutdownController.forceStop(ManagementShutdownController.java:63)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stopImpl(LocalLegacyShutdownController.java:120)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:66)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.ide.eclipse.as.wtp.core.server.behavior.ControllableServerBehavior.stop(ControllableServerBehavior.java:255)
	at org.eclipse.wst.server.core.internal.Server.stopImpl2(Server.java:3688)
	at org.eclipse.wst.server.core.internal.Server.stopImpl(Server.java:3645)
	at org.eclipse.wst.server.core.internal.Server$StopJob.run(Server.java:403)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:149)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:75)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:350)
	... 24 more
Caused by: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:122)
	at org.jboss.as.protocol.ProtocolConnectionManager$EstablishingConnection.connect(ProtocolConnectionManager.java:257)
	at org.jboss.as.protocol.ProtocolConnectionManager.connect(ProtocolConnectionManager.java:71)
	at org.jboss.as.protocol.mgmt.FutureManagementChannel$Establishing.getChannel(FutureManagementChannel.java:212)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient.getOrCreateChannel(RemotingModelControllerClient.java:146)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient$1.getChannel(RemotingModelControllerClient.java:65)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:147)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:122)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeRequest(AbstractModelControllerClient.java:263)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:168)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:147)
	... 26 more
Caused by: java.io.IOException: Invalid response
	at org.xnio.http.HttpUpgradeParser.parseVersion(HttpUpgradeParser.java:157)
	at org.xnio.http.HttpUpgradeParser.parse(HttpUpgradeParser.java:55)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:419)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:400)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:88)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:559)
	at ...asynchronous invocation...(Unknown Source)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:294)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:275)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:379)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:367)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connect(ProtocolConnectionUtils.java:83)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:114)
	... 36 more

!ENTRY org.jboss.ide.eclipse.as.core 4 16973826 2017-01-09 11:14:03.452
!MESSAGE The server was shutdown forcefully. All processes were terminated.

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:14:07.441
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:14:07.507
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:14:19.925
!MESSAGE The server "WildFly 10.x" requires a JDK to run properly. The VM used to launch this server is not recognized as a valid JDK. C:\Program Files\Java\jre1.8.0_111 

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:14:33.252
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:14:33.264
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:18:25.153
!MESSAGE Unable to retrieve a list of remote deployment scanners
!STACK 0
org.jboss.ide.eclipse.as.management.core.JBoss7ManangerException: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:361)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9ManagerService.execute(Wildfly9ManagerService.java:171)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerServiceProxy.execute(JBoss7ManagerServiceProxy.java:87)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:294)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:1)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerUtil.executeWithService(JBoss7ManagerUtil.java:136)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.executeWithResult(AS7DeploymentScannerUtility.java:292)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScanners(AS7DeploymentScannerUtility.java:163)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScannersFromServer(AS7DeploymentScannerUtility.java:252)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServer(LocalJBoss7DeploymentScannerAdditions.java:173)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServerSafely(LocalJBoss7DeploymentScannerAdditions.java:135)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.ensureScannersRemoved(LocalJBoss7DeploymentScannerAdditions.java:150)
	at org.jboss.ide.eclipse.as.core.server.internal.AbstractDeploymentScannerAdditions.removeAddedDeploymentScanners(AbstractDeploymentScannerAdditions.java:82)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.removeScanners(LocalLegacyShutdownController.java:75)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:58)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.ide.eclipse.as.wtp.core.server.behavior.ControllableServerBehavior.stop(ControllableServerBehavior.java:255)
	at org.eclipse.wst.server.core.internal.Server.stopImpl2(Server.java:3688)
	at org.eclipse.wst.server.core.internal.Server.stopImpl(Server.java:3645)
	at org.eclipse.wst.server.core.internal.Server$StopJob.run(Server.java:403)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:149)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:75)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:350)
	... 20 more
Caused by: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:122)
	at org.jboss.as.protocol.ProtocolConnectionManager$EstablishingConnection.connect(ProtocolConnectionManager.java:257)
	at org.jboss.as.protocol.ProtocolConnectionManager.connect(ProtocolConnectionManager.java:71)
	at org.jboss.as.protocol.mgmt.FutureManagementChannel$Establishing.getChannel(FutureManagementChannel.java:212)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient.getOrCreateChannel(RemotingModelControllerClient.java:146)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient$1.getChannel(RemotingModelControllerClient.java:65)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:147)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:122)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeRequest(AbstractModelControllerClient.java:263)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:168)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:147)
	... 22 more
Caused by: java.io.IOException: Invalid response
	at org.xnio.http.HttpUpgradeParser.parseVersion(HttpUpgradeParser.java:157)
	at org.xnio.http.HttpUpgradeParser.parse(HttpUpgradeParser.java:55)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:419)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:400)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:88)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:559)
	at ...asynchronous invocation...(Unknown Source)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:294)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:275)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:379)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:367)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connect(ProtocolConnectionUtils.java:83)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:114)
	... 32 more

!ENTRY org.eclipse.wst.server.core 4 0 2017-01-09 11:18:25.230
!MESSAGE Server WildFly 10.x failed to stop.

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:18:29.726
!MESSAGE Unable to retrieve a list of remote deployment scanners
!STACK 0
org.jboss.ide.eclipse.as.management.core.JBoss7ManangerException: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:361)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9ManagerService.execute(Wildfly9ManagerService.java:171)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerServiceProxy.execute(JBoss7ManagerServiceProxy.java:87)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:294)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:1)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerUtil.executeWithService(JBoss7ManagerUtil.java:136)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.executeWithResult(AS7DeploymentScannerUtility.java:292)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScanners(AS7DeploymentScannerUtility.java:163)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScannersFromServer(AS7DeploymentScannerUtility.java:252)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServer(LocalJBoss7DeploymentScannerAdditions.java:173)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServerSafely(LocalJBoss7DeploymentScannerAdditions.java:135)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.ensureScannersRemoved(LocalJBoss7DeploymentScannerAdditions.java:150)
	at org.jboss.ide.eclipse.as.core.server.internal.AbstractDeploymentScannerAdditions.removeAddedDeploymentScanners(AbstractDeploymentScannerAdditions.java:82)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.removeScanners(LocalLegacyShutdownController.java:75)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:58)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.ide.eclipse.as.wtp.core.server.behavior.ControllableServerBehavior.stop(ControllableServerBehavior.java:255)
	at org.eclipse.wst.server.core.internal.Server.stopImpl2(Server.java:3688)
	at org.eclipse.wst.server.core.internal.Server.stopImpl(Server.java:3645)
	at org.eclipse.wst.server.core.internal.Server$StopJob.run(Server.java:403)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:149)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:75)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:350)
	... 20 more
Caused by: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:122)
	at org.jboss.as.protocol.ProtocolConnectionManager$EstablishingConnection.connect(ProtocolConnectionManager.java:257)
	at org.jboss.as.protocol.ProtocolConnectionManager.connect(ProtocolConnectionManager.java:71)
	at org.jboss.as.protocol.mgmt.FutureManagementChannel$Establishing.getChannel(FutureManagementChannel.java:212)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient.getOrCreateChannel(RemotingModelControllerClient.java:146)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient$1.getChannel(RemotingModelControllerClient.java:65)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:147)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:122)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeRequest(AbstractModelControllerClient.java:263)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:168)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:147)
	... 22 more
Caused by: java.io.IOException: Invalid response
	at org.xnio.http.HttpUpgradeParser.parseVersion(HttpUpgradeParser.java:157)
	at org.xnio.http.HttpUpgradeParser.parse(HttpUpgradeParser.java:55)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:419)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:400)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:88)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:559)
	at ...asynchronous invocation...(Unknown Source)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:294)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:275)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:379)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:367)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connect(ProtocolConnectionUtils.java:83)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:114)
	... 32 more

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:18:29.796
!MESSAGE Unable to retrieve a list of remote deployment scanners
!STACK 0
org.jboss.ide.eclipse.as.management.core.JBoss7ManangerException: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:361)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9ManagerService.execute(Wildfly9ManagerService.java:171)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerServiceProxy.execute(JBoss7ManagerServiceProxy.java:87)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:294)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:1)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerUtil.executeWithService(JBoss7ManagerUtil.java:136)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.executeWithResult(AS7DeploymentScannerUtility.java:292)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScanners(AS7DeploymentScannerUtility.java:163)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScannersFromServer(AS7DeploymentScannerUtility.java:252)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServer(LocalJBoss7DeploymentScannerAdditions.java:173)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServerSafely(LocalJBoss7DeploymentScannerAdditions.java:135)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.ensureScannersRemoved(LocalJBoss7DeploymentScannerAdditions.java:150)
	at org.jboss.ide.eclipse.as.core.server.internal.AbstractDeploymentScannerAdditions.removeAddedDeploymentScanners(AbstractDeploymentScannerAdditions.java:82)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.removeScanners(LocalLegacyShutdownController.java:75)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:58)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.ManagementShutdownController.forceStop(ManagementShutdownController.java:63)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stopImpl(LocalLegacyShutdownController.java:120)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:66)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.ide.eclipse.as.wtp.core.server.behavior.ControllableServerBehavior.stop(ControllableServerBehavior.java:255)
	at org.eclipse.wst.server.core.internal.Server.stopImpl2(Server.java:3688)
	at org.eclipse.wst.server.core.internal.Server.stopImpl(Server.java:3645)
	at org.eclipse.wst.server.core.internal.Server$StopJob.run(Server.java:403)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:149)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:75)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:350)
	... 24 more
Caused by: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:122)
	at org.jboss.as.protocol.ProtocolConnectionManager$EstablishingConnection.connect(ProtocolConnectionManager.java:257)
	at org.jboss.as.protocol.ProtocolConnectionManager.connect(ProtocolConnectionManager.java:71)
	at org.jboss.as.protocol.mgmt.FutureManagementChannel$Establishing.getChannel(FutureManagementChannel.java:212)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient.getOrCreateChannel(RemotingModelControllerClient.java:146)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient$1.getChannel(RemotingModelControllerClient.java:65)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:147)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:122)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeRequest(AbstractModelControllerClient.java:263)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:168)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:147)
	... 26 more
Caused by: java.io.IOException: Invalid response
	at org.xnio.http.HttpUpgradeParser.parseVersion(HttpUpgradeParser.java:157)
	at org.xnio.http.HttpUpgradeParser.parse(HttpUpgradeParser.java:55)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:419)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:400)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:88)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:559)
	at ...asynchronous invocation...(Unknown Source)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:294)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:275)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:379)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:367)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connect(ProtocolConnectionUtils.java:83)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:114)
	... 36 more

!ENTRY org.jboss.ide.eclipse.as.core 4 16973826 2017-01-09 11:18:29.818
!MESSAGE The server was shutdown forcefully. All processes were terminated.

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:18:33.181
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:18:33.195
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:18:45.302
!MESSAGE The server "WildFly 10.x" requires a JDK to run properly. The VM used to launch this server is not recognized as a valid JDK. C:\Program Files\Java\jre1.8.0_111 

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:18:58.273
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:18:58.334
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY tern.eclipse.ide.core 4 0 2017-01-09 11:21:41.503
!MESSAGE TernError: No type found at the given position.
!STACK 0
tern.TernException: TernError: No type found at the given position.
	at tern.server.DefaultResponseHandler.getData(DefaultResponseHandler.java:45)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:43)
	at tern.eclipse.ide.internal.core.resources.IDETernServerAsyncReqProcessor.run(IDETernServerAsyncReqProcessor.java:104)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: tern.TernException: TernError: No type found at the given position.
	at tern.server.nodejs.NodejsTernHelper.makeRequest(NodejsTernHelper.java:86)
	at tern.server.nodejs.NodejsTernServer.makeRequest(NodejsTernServer.java:163)
	at tern.server.nodejs.NodejsTernServer.request(NodejsTernServer.java:127)
	at tern.server.protocol.TernResultsProcessorsFactory.makeRequestAndProcess(TernResultsProcessorsFactory.java:42)
	... 2 more

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:21:45.435
!MESSAGE Unable to retrieve a list of remote deployment scanners
!STACK 0
org.jboss.ide.eclipse.as.management.core.JBoss7ManangerException: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:361)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9ManagerService.execute(Wildfly9ManagerService.java:171)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerServiceProxy.execute(JBoss7ManagerServiceProxy.java:87)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:294)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:1)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerUtil.executeWithService(JBoss7ManagerUtil.java:136)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.executeWithResult(AS7DeploymentScannerUtility.java:292)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScanners(AS7DeploymentScannerUtility.java:163)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScannersFromServer(AS7DeploymentScannerUtility.java:252)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServer(LocalJBoss7DeploymentScannerAdditions.java:173)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServerSafely(LocalJBoss7DeploymentScannerAdditions.java:135)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.ensureScannersRemoved(LocalJBoss7DeploymentScannerAdditions.java:150)
	at org.jboss.ide.eclipse.as.core.server.internal.AbstractDeploymentScannerAdditions.removeAddedDeploymentScanners(AbstractDeploymentScannerAdditions.java:82)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.removeScanners(LocalLegacyShutdownController.java:75)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:58)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.ide.eclipse.as.wtp.core.server.behavior.ControllableServerBehavior.stop(ControllableServerBehavior.java:255)
	at org.eclipse.wst.server.core.internal.Server.stopImpl2(Server.java:3688)
	at org.eclipse.wst.server.core.internal.Server.stopImpl(Server.java:3645)
	at org.eclipse.wst.server.core.internal.Server$StopJob.run(Server.java:403)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:149)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:75)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:350)
	... 20 more
Caused by: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:122)
	at org.jboss.as.protocol.ProtocolConnectionManager$EstablishingConnection.connect(ProtocolConnectionManager.java:257)
	at org.jboss.as.protocol.ProtocolConnectionManager.connect(ProtocolConnectionManager.java:71)
	at org.jboss.as.protocol.mgmt.FutureManagementChannel$Establishing.getChannel(FutureManagementChannel.java:212)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient.getOrCreateChannel(RemotingModelControllerClient.java:146)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient$1.getChannel(RemotingModelControllerClient.java:65)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:147)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:122)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeRequest(AbstractModelControllerClient.java:263)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:168)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:147)
	... 22 more
Caused by: java.io.IOException: Invalid response
	at org.xnio.http.HttpUpgradeParser.parseVersion(HttpUpgradeParser.java:157)
	at org.xnio.http.HttpUpgradeParser.parse(HttpUpgradeParser.java:55)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:419)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:400)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:88)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:559)
	at ...asynchronous invocation...(Unknown Source)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:294)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:275)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:379)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:367)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connect(ProtocolConnectionUtils.java:83)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:114)
	... 32 more

!ENTRY org.eclipse.wst.server.core 4 0 2017-01-09 11:21:45.513
!MESSAGE Server WildFly 10.x failed to stop.

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:21:50.379
!MESSAGE Unable to retrieve a list of remote deployment scanners
!STACK 0
org.jboss.ide.eclipse.as.management.core.JBoss7ManangerException: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:361)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9ManagerService.execute(Wildfly9ManagerService.java:171)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerServiceProxy.execute(JBoss7ManagerServiceProxy.java:87)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:294)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:1)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerUtil.executeWithService(JBoss7ManagerUtil.java:136)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.executeWithResult(AS7DeploymentScannerUtility.java:292)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScanners(AS7DeploymentScannerUtility.java:163)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScannersFromServer(AS7DeploymentScannerUtility.java:252)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServer(LocalJBoss7DeploymentScannerAdditions.java:173)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServerSafely(LocalJBoss7DeploymentScannerAdditions.java:135)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.ensureScannersRemoved(LocalJBoss7DeploymentScannerAdditions.java:150)
	at org.jboss.ide.eclipse.as.core.server.internal.AbstractDeploymentScannerAdditions.removeAddedDeploymentScanners(AbstractDeploymentScannerAdditions.java:82)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.removeScanners(LocalLegacyShutdownController.java:75)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:58)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.ide.eclipse.as.wtp.core.server.behavior.ControllableServerBehavior.stop(ControllableServerBehavior.java:255)
	at org.eclipse.wst.server.core.internal.Server.stopImpl2(Server.java:3688)
	at org.eclipse.wst.server.core.internal.Server.stopImpl(Server.java:3645)
	at org.eclipse.wst.server.core.internal.Server$StopJob.run(Server.java:403)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:149)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:75)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:350)
	... 20 more
Caused by: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:122)
	at org.jboss.as.protocol.ProtocolConnectionManager$EstablishingConnection.connect(ProtocolConnectionManager.java:257)
	at org.jboss.as.protocol.ProtocolConnectionManager.connect(ProtocolConnectionManager.java:71)
	at org.jboss.as.protocol.mgmt.FutureManagementChannel$Establishing.getChannel(FutureManagementChannel.java:212)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient.getOrCreateChannel(RemotingModelControllerClient.java:146)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient$1.getChannel(RemotingModelControllerClient.java:65)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:147)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:122)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeRequest(AbstractModelControllerClient.java:263)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:168)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:147)
	... 22 more
Caused by: java.io.IOException: Invalid response
	at org.xnio.http.HttpUpgradeParser.parseVersion(HttpUpgradeParser.java:157)
	at org.xnio.http.HttpUpgradeParser.parse(HttpUpgradeParser.java:55)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:419)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:400)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:88)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:559)
	at ...asynchronous invocation...(Unknown Source)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:294)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:275)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:379)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:367)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connect(ProtocolConnectionUtils.java:83)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:114)
	... 32 more

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:21:50.468
!MESSAGE Unable to retrieve a list of remote deployment scanners
!STACK 0
org.jboss.ide.eclipse.as.management.core.JBoss7ManangerException: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:361)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9ManagerService.execute(Wildfly9ManagerService.java:171)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerServiceProxy.execute(JBoss7ManagerServiceProxy.java:87)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:294)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility$1.execute(AS7DeploymentScannerUtility.java:1)
	at org.jboss.ide.eclipse.as.management.core.JBoss7ManagerUtil.executeWithService(JBoss7ManagerUtil.java:136)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.executeWithResult(AS7DeploymentScannerUtility.java:292)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScanners(AS7DeploymentScannerUtility.java:163)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.AS7DeploymentScannerUtility.getDeploymentScannersFromServer(AS7DeploymentScannerUtility.java:252)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServer(LocalJBoss7DeploymentScannerAdditions.java:173)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.loadScannersFromServerSafely(LocalJBoss7DeploymentScannerAdditions.java:135)
	at org.jboss.ide.eclipse.as.core.server.internal.v7.LocalJBoss7DeploymentScannerAdditions.ensureScannersRemoved(LocalJBoss7DeploymentScannerAdditions.java:150)
	at org.jboss.ide.eclipse.as.core.server.internal.AbstractDeploymentScannerAdditions.removeAddedDeploymentScanners(AbstractDeploymentScannerAdditions.java:82)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.removeScanners(LocalLegacyShutdownController.java:75)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:58)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.ManagementShutdownController.forceStop(ManagementShutdownController.java:63)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stopImpl(LocalLegacyShutdownController.java:120)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:66)
	at org.jboss.tools.as.core.server.controllable.subsystems.internal.LocalLegacyShutdownController.stop(LocalLegacyShutdownController.java:53)
	at org.jboss.ide.eclipse.as.wtp.core.server.behavior.ControllableServerBehavior.stop(ControllableServerBehavior.java:255)
	at org.eclipse.wst.server.core.internal.Server.stopImpl2(Server.java:3688)
	at org.eclipse.wst.server.core.internal.Server.stopImpl(Server.java:3645)
	at org.eclipse.wst.server.core.internal.Server$StopJob.run(Server.java:403)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:55)
Caused by: java.io.IOException: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:149)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:75)
	at org.jboss.ide.eclipse.as.internal.management.wildfly9.Wildfly9Manager.execute(Wildfly9Manager.java:350)
	... 24 more
Caused by: java.net.ConnectException: WFLYPRT0053: Could not connect to http-remoting://localhost:9990. The connection failed
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:122)
	at org.jboss.as.protocol.ProtocolConnectionManager$EstablishingConnection.connect(ProtocolConnectionManager.java:257)
	at org.jboss.as.protocol.ProtocolConnectionManager.connect(ProtocolConnectionManager.java:71)
	at org.jboss.as.protocol.mgmt.FutureManagementChannel$Establishing.getChannel(FutureManagementChannel.java:212)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient.getOrCreateChannel(RemotingModelControllerClient.java:146)
	at org.jboss.as.controller.client.impl.RemotingModelControllerClient$1.getChannel(RemotingModelControllerClient.java:65)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:147)
	at org.jboss.as.protocol.mgmt.ManagementChannelHandler.executeRequest(ManagementChannelHandler.java:122)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeRequest(AbstractModelControllerClient.java:263)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.execute(AbstractModelControllerClient.java:168)
	at org.jboss.as.controller.client.impl.AbstractModelControllerClient.executeForResult(AbstractModelControllerClient.java:147)
	... 26 more
Caused by: java.io.IOException: Invalid response
	at org.xnio.http.HttpUpgradeParser.parseVersion(HttpUpgradeParser.java:157)
	at org.xnio.http.HttpUpgradeParser.parse(HttpUpgradeParser.java:55)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:419)
	at org.xnio.http.HttpUpgrade$HttpUpgradeState$UpgradeResultListener.handleEvent(HttpUpgrade.java:400)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.conduits.ReadReadyHandler$ChannelListenerHandler.readReady(ReadReadyHandler.java:66)
	at org.xnio.nio.NioSocketConduit.handleReady(NioSocketConduit.java:88)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:559)
	at ...asynchronous invocation...(Unknown Source)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:294)
	at org.jboss.remoting3.EndpointImpl.doConnect(EndpointImpl.java:275)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:379)
	at org.jboss.remoting3.EndpointImpl.connect(EndpointImpl.java:367)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connect(ProtocolConnectionUtils.java:83)
	at org.jboss.as.protocol.ProtocolConnectionUtils.connectSync(ProtocolConnectionUtils.java:114)
	... 36 more

!ENTRY org.jboss.ide.eclipse.as.core 4 16973826 2017-01-09 11:21:50.499
!MESSAGE The server was shutdown forcefully. All processes were terminated.

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:21:53.551
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:21:53.605
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:22:06.370
!MESSAGE The server "WildFly 10.x" requires a JDK to run properly. The VM used to launch this server is not recognized as a valid JDK. C:\Program Files\Java\jre1.8.0_111 

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:22:19.635
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x

!ENTRY org.jboss.ide.eclipse.as.core 4 0 2017-01-09 11:22:19.673
!MESSAGE Unable to retrieve a list of remote deployment scanners for server WildFly 10.x
